marin/resources.py

```diff
@@ -80,7 +117,25 @@ class GpuConfig(ResourceConfig):
     runtime_env: RuntimeEnv = dataclasses.field(default_factory=RuntimeEnv)
 
     accelerator_type: AcceleratorType | None = None
-    """Type of GPU accelerator to use. If None, will use any available GPU."""
+    """Type of GPU to use. Must be one of the Ray accelerator types."""
```

Comment:
give an 2 examples (e.g., a100, v4-128) or whatever

Reply:
also just link to the list